cli_tools_tests/module/diskinspect/disk_inspect_test.go

```diff
@@ -75,6 +79,74 @@ func TestBootInspect(t *testing.T) {
 				Distro:       "windows",
 				Major:        "6",
 				Minor:        "1",
+				UEFIBootable: false,
+				BIOSBootable: false,
+			},
+		}, {
+			// UEFI inspection test for GPT UEFI
+			"projects/gce-uefi-images/global/images/rhel-7-v20200403",
+			disk.InspectionResult{
+				Architecture: "x64",
+				Distro:       "rhel",
+				Major:        "7",
+				Minor:        "8",
+				UEFIBootable: true,
+				BIOSBootable: false,
+			},
+		}, {
+			// UEFI inspection test for MBR-only
+			"projects/debian-cloud/global/images/debian-9-stretch-v20200714",
+			disk.InspectionResult{
+				Architecture: "x64",
+				Distro:       "debian",
+				Major:        "9",
+				Minor:        "12",
+				UEFIBootable: false,
+				BIOSBootable: false,
```

Comment:
When I read this I think: "The disk isn't bootable, since it can't boot with UEFI and it can't boot with BIOS."

I propose two options:

1. Update your detection to determine if a disk is bootable with MBR and BIOS 
2. Split `BIOSBootable` into two variables, documenting that a disk *may* be bootable, even when all three are false
   * GPT BIOS bootable
   * Hybrid MBR bootable

Reply:
renamed to BIOSBootableWithHybridMBROrProtectiveMBR for better understandable. We don't need to split since we won't treat them differently.